parley/src/layout/editor.rs

```diff
@@ -49,7 +51,9 @@ where
             selection: Default::default(),
             cursor_mode: Default::default(),
             width: Default::default(),
-            scale: Default::default(),
+            scale: 1.0,
+            dirty_layout: Default::default(),
```

Comment:
I'd probably call this `layout_dirty`; this way around reads a bit strangely to me.

Reply:
Fair enough, sounds like a verb.